modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase

```diff
@@ -1185,6 +1186,113 @@ definition {
 		AppBuilderAdmin.deleteEntry(itemName = "my custom object");
 	}
 
+	@description = "Delete fields from the Form View through the Delete options and assert its affect on other Form and Table views"
+	@priority = "5"
+	test DeleteFormViewFieldFromObject {
+
+		AppBuilderAdmin.addCustomObject(
+			customObjectName = "my custom object",
+			isCreatingFormView = "true",
+			isCreatingViaPlusIcon = "true");
+
+		FormViewBuilder.setName(formViewName = "My Form View");
+
+		FormViewBuilder.addFieldByDoubleClick(fieldType = "Text");
+
+		FormViewBuilder.closeSidebar();
+
+		FormViewBuilder.addFieldByDoubleClick(fieldType = "Select from List");
+
+		FormViewBuilder.save();
```

Comment:
Hi @tainguyen100 , I would suggest you to create another Form View (My Form View 2), so you can remove some fields from the Object through the edition of Form View 1 as you are doing, and then you can validate if the deleted fields were removed from the Object sidebar at Form View 2.